### PR TITLE
Update in ResourceManagement

### PR DESCRIPTION
**Pull request by samigeorgiev**

### CODE REVIEW

File: client/src/components/lecture/CreateLecture.jsx [hunk 1]
```diff
@@ -1,10 +1,37 @@
...
+        const request = new CreateLectureRequest();
+        request.setCourseid(course);
+        request.setName(inputsValues.name);
+        createLecture(request, token);
+    };
+
+    const { response, error } = state;
+    useEffect(() => {
+        if (response && !error) {
+            console.log(response);
```
Comment (toniyanchev) [inline client/src/components/lecture/CreateLecture.jsx:31]: why u log?

File: client/src/components/lecture/LectureEditList.jsx [hunk 2]
```diff
@@ -73,6 +73,7 @@ const LectureEditList = props => {
         <Modal onClose={props.onClose} centered={false} open closeIcon>
```
Comment (toniyanchev) [inline client/src/components/lecture/LectureEditList.jsx:73]: better on new lines

File: client/src/components/lecture/CreateLecture.jsx (hunk 1 above)
Comment (samigeorgiev) [reply client/src/components/lecture/CreateLecture.jsx:31]: after your fix, here will be the logic for uploading video

File: client/src/components/lecture/LectureEditList.jsx (hunk 2 above)
Comment (samigeorgiev) [reply client/src/components/lecture/LectureEditList.jsx:73]: prettier formats it, not me

**Pull request merged**